README.md

```diff
@@ -31,31 +31,37 @@ Add below content:
 127.0.0.1 prod.foo.redhat.com
 ```
 
+OR
+
+run the script (with sudo)
+
+```
+npm run patch:hosts
```

Comment:
What does this script do? Could you add something as a comment?

Reply:
@r14chandra it will Setup the initial /etc/hosts entries as mentioned above in the step 2. It is an alternate way. Can you go through step 2 again and let me know if it is still not clear then I will make further changes to it. Thanks!